examples/window-app/nrfconnect/overlay-low_power.conf

```diff
@@ -17,6 +17,9 @@
 # Enable MTD Sleepy End Device
 CONFIG_CHIP_ENABLE_SLEEPY_END_DEVICE_SUPPORT=y
 
+# Enable Thread CSL to support Synchronized SED
+CONFIG_CHIP_THREAD_SSED=y
```

Comment:
In the readme there is a part describing that using this overlay enables Thread Sleepy End Device. If we are going to change that to SSED, I think we should also update the readme.

Reply:
Fixed by rebase.